ingestion/src/metadata/profiler/metrics/static/mean.py

```diff
@@ -121,7 +121,22 @@ def df_fn(self, dfs=None):
 
         if is_quantifiable(self.col.type):
             for df in dfs:
-                mean = df[self.col.name].mean()
+                try:
+                    mean = df[self.col.name].mean()
+                except Exception as err:
+                    if str(err).startswith("Could not convert") and str(err).endswith(
+                        "to numeric"
+                    ):
+                        [
+                            float(item)
+                            for item in df[self.col.name].values
+                            if item.isdigit()
+                        ]
+                        mean = None
```

Comment:
what are we doing here? we convert all the values to `float` and then set the mean to None? I think if we can't compute the mean better not computing it at all. Otherwise it is going to be super costly with very large data set to iterate and convert every value

Reply:
removed the logic of assigning type as float here

Reply:
reverted it back, the code was the before the discussion around evaluating the literal datatypes..